pinot-spi/src/main/java/org/apache/pinot/spi/utils/PinotReflectionUtils.java

```diff
@@ -53,7 +53,7 @@ public static Set<Class<?>> getClassesThroughReflection(String packageName, Stri
     try {
       synchronized (REFLECTION_LOCK) {
         return new Reflections(new ConfigurationBuilder().setUrls(ClasspathHelper.forPackage(packageName))
-            .filterInputsBy(new FilterBuilder().includePattern(regexPattern))).getTypesAnnotatedWith(annotation);
+            .filterInputsBy(new FilterBuilder.Include(regexPattern))).getTypesAnnotatedWith(annotation);
```

Comment:
I think it will work for both versions if you change this to
```suggestion
            .filterInputsBy(new FilterBuilder().include(regexPattern))).getTypesAnnotatedWith(annotation);
```

Reply:
Good point, thanks. I have made the changes and build succeeded with 0.9.11.